### DISCUSSION TITLE
Maven publishing discontinued?

### DISCUSSION THREAD

**Discussion by hmf** [HanSolo/charts#47] [Q&A] [answered]
Hi,

I see that the latest version in  Maven is of [version 11.0](https://search.maven.org/artifact/eu.hansolo.fx.charts/charts) (June, 2020). But the [latest release](https://github.com/HanSolo/charts/releases) is 11.7 (April, 2021). Has the publishing of the library to the repository been discontinued?

TIA

**Comment by HanSolo** [comment, marked as answer]
You can find them here: https://search.maven.org/artifact/eu.hansolo.fx/charts 
And nothing has been discontinued, just released 16.0.5 today.

**Comment by hmf** [comment]
@HanSolo My aoplogies but the search URL I use does not show all the published versions. Even now I see only 2 versions. But your URL show several others. Strange. May be something to do with the [artifact ids](https://search.maven.org/search?q=eu.hansolo.fx). 

Thanks.

**Reply by HanSolo** [reply]
No problem, indeed I've changed the artifact id at one point and forgot to mention it, glad you found them now 👍🏻
